### PR TITLE
Add 10.13 targeting

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
@maririos PTAL.

File: jobs/generation/Utilities.groovy [hunk 1]
```diff
@@ -189,15 +189,6 @@ class Utilities {
                                 // For outerloop runs.
                                 'outer-latest-or-auto':'ubuntu1610-20170216-outer',
                                 ],
-                            'OSX10.11' :
-                                [
-                                // Generic version label
-                                '' : 'mac',
```
Comment (maririos) [inline jobs/generation/Utilities.groovy]: I imagine no one is using this one, so we are not going to brake them with this change, right?

Comment (mmitche) [inline jobs/generation/Utilities.groovy]: there were two element in the switch statement that had 10.11. I have no idea that was even possible.

**Comment by mmitche** [comment]
@maririos You can actually tell whether you're going to break someone by going to:

https://ci.dot.net/label/<label name>/

It will show the projects + machines associated with the label.

File: jobs/generation/Utilities.groovy (hunk 1 above)
Comment (maririos) [inline jobs/generation/Utilities.groovy]: haha :|

**Comment by maririos** [comment]
ohh nice. LGTM

**Pull request merged**